[PATCH] fix alsa version check

Check the entire alsa version, and not just the subminor - As this broke
with the recent alsa 1.0.x -> 1.1.0 release.

It is not clear what major/minor versions the check was supposed to be
against (0.9 or 1.0).  I went with 1.0.11 to be on the safe side.

Signed-off-by: Peter Korsgaard <peter@korsgaard.com>
---
 flite-1.4-release/configure.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: flite-1.4/flite-1.4-release/configure.in
===================================================================
--- flite-1.4.orig/flite-1.4-release/configure.in
+++ flite-1.4/flite-1.4-release/configure.in
@@ -270,7 +270,7 @@
 # the one I know -- you can still specific --with-audio=alsa
 AC_TRY_COMPILE([#include <alsa/version.h>],
                [int j=
-                #if SND_LIB_SUBMINOR >= 11
+                #if SND_LIB_VERSION >= 0x1000b
                 3;
                 #endif],
               [AUDIODRIVER="alsa"
